android/app/src/main/java/org/bitcoinppl/cove/flows/SendFlow/EnterAddressView.kt

```
@@ -105,57 +143,183 @@ fun EnterAddressView(
+                        modifier = Modifier.weight(1f)
+                    )
+                    TextButton(onClick = { showingWalletPicker = false }) {
+                        Text("Cancel", color = CoveColor.LinkBlue)
+                    }
+                }
+                Spacer(Modifier.height(16.dp))
+
+                val wallets = remember {
+                    try {
+                        Database().wallets().allSortedActive().filter { it.id != currentWalletId }
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)